### ISSUE TITLE
Help Please

### ISSUE THREAD

**Issue by rickman297** [OSVR/SteamVR-OSVR#38]
Someone make a video on this please. I'm a visual leaner and I need someone to dumb it down for me. This is too complicated for me and i just need someone to send me a link on a video on how to do this. I just want to play The Arrival on the OSVR. Thank you.

**Comment by godbyk** [comment]
@rickman297 Which operating system are you using (Windows, Linux, OS X)? What installation steps have you taken so far?

**Comment by rickman297** [comment]
im using windows 10 and i have downloaded the osvr server, ir tracking, i can get regular osvr demos fine. i downloaded this and their is no file that says "driver" in it.

**Comment by fieldy1** [comment]
Follow this link -https://bintray.com/osvr/SteamVR-OSVR/SteamVR-OSVR-Win/v0.1-81-g2862d60-core-v0.6-1074-g11272ab/view#files

**Comment by godbyk** [comment]
I'm closing this issue due to a lack of information. If you are the original filer of this issue and still encounter this problem, feel free to reopen this issue. If you're not the original filer of this issue, please open a new issue instead.
